src/datasources/cache/__tests__/fake.cache.service.spec.ts

```diff
@@ -8,6 +8,7 @@ describe('FakeCacheService', () => {
 
   beforeEach(async () => {
     target = new FakeCacheService();
+    jest.useRealTimers();
```

Comment:
Since this was done because of the change of a single test I would move this to that test wdyt?

Reply:
Done in [b92584c](https://github.com/safe-global/safe-client-gateway/pull/970/commits/b92584cf5c50470b34305c8c9afffcc884c9fb21)